src/components/project/Project.vue

```diff
@@ -1,5 +1,10 @@
 <template>
+    <project-drawer
+        v-model="selectedPage"
+    />
+
     <v-navigation-drawer
+        v-if="selectedPage === ProjectDrawerItem.ANALYSIS"
```

Comment:
I feel like we should be able to replace this navigation with a subrouter here. That way, we can also use URLs like `/mpa/export`, `/mpa/databases`, ...